includes/class-wp-job-manager-ajax.php

```diff
@@ -185,7 +185,13 @@ public function get_listings() {
 			$result['showing_all'] = true;
 		}
 
-		$result['showing'] = apply_filters( 'job_manager_get_listings_custom_filter_text', sprintf( __( 'Showing all %s', 'wp-job-manager' ), implode( ' ', $result['showing'] ) ) );
+		if ( $jobs->post_count && ( $search_location || $search_keywords || $search_categories ) ) {
+			$message = sprintf( __( '%1$s search completed. Found %2$d matching records.', 'wp-job-manager' ), __( $post_type_label, 'wp-job-manager' ), $jobs->post_count );
```

Comment:
`xgettext` will probably choke on this, as it extracts translatable strings from plain text. What about changing this to:
if ( 'Jobs' === $post_type_label ) {
$message = sprintf( _n( 'Jobs search completed. Found %d matching record.', 'Jobs search completed. Found %d matching records.', $jobs->post_count, 'wp-job-manager' ), $jobs->post_count );
} else {
$message = sprintf( _n( 'Search completed. Found %d matching record.', 'Jobs search completed. Found %d matching records.', $jobs->post_count, 'wp-job-manager' ), $jobs->post_count );
}